wtf/utils.go

```diff
@@ -149,7 +149,7 @@ func ToStrs(slice []interface{}) []string {
 	results := []string{}
 
 	for _, val := range slice {
-		results = append(results, val.(string))
+		results = append(results, fmt.Sprintf("%v", val))
```

Comment:
This exists because I was getting a type assertion error for turning an int (todoist projects are ints) into a string. Since `%v` prints the object value, this seemed the easiest thing to do to handle strings and ints without error handling or reflection.

Reply:
Something like this should work and is extensible for other types should they come up:
```
for _, val := range slice {
		switch val.(type) {
		case int:
			results = append(results, strconv.Itoa(val.(int)))
		case string:
			results = append(results, val.(string))
		}
	}
```